code/style.css

```diff
@@ -1,5 +1,279 @@
+  width: auto; /* Mantiene la proporción de la imagen */
+}
+
+nav ul {
+  list-style-type:none;
+  margin: 0%;
+  padding: 3;
+  display:flex;
+  color: rgb(246, 248, 248);
+}
+
+nav ul li {
+  float: right;
+}
+
+nav ul li a {
+  display: block;
+  color: #5358F6;
+  text-align: last;
+  padding: 20px 20px;
+  text-decoration: none;
+}
+
+nav ul li a:hover {
+  background-color: #ddd;
+  
+}
+
+.hero-container {
+  position: relative;
+  width: 100%;
+  height: 100vh; /* 50% de la altura de la ventana gráfica */
+  overflow: hidden;
+}
+
+#hero-video {
+  position:absolute;
+  top: 50%;
+  left: 50%;
+  min-width: 100%;
+  min-height: 100%;
+  width: auto;
+  height: auto;
+  transform: translate(-50%, -50%); /* Centra el video */
+  z-index: -1; /* Envía el video al fondo */
+}
+
+.hero-text {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%); /* Centra el texto */
+  text-align: center;
+  z-index: 1; /* Asegura que el texto esté por encima del video */
+}
+
+.hero-text h1 {
+  font-size: 48px;
+  margin: 0;
+  color: #FDFD7E;
+}
+
+.hero-text p {
+  font-size: 24px;
+  margin: 10px 0;
+  color: #ddd;
+}
+
+.hero-text button {
+  padding: 10px 20px;
+  font-size: 18px;
+  background-color: rgb(109, 106, 106) ; /* Color dorado para el botón */
+  border: none;
+  color: #FDFD7E;
+  cursor: pointer;
+  transition: background-color 0.3s ease;
+}
+
+.hero-text button:hover {
+  background-color: #d8d8d4; /* Color más oscuro al pasar el ratón */
+}
+
+/*images container*/
+
+.image-gallery {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin-top: 80px;
+  margin-bottom: 80px;
+  gap: 100px;
+  transition: .5s ease;
+  backface-visibility: hidden;
+}
+
+/*Default styles for larger screens*/
+.image-gallery a:nth-child(3) {
+  flex: 1 1 1 100%; /*Default to 3 images per row*/
+  display: flex;
+  justify-content: center;
+ }
+
+
+.image-gallery img {
+  border: 1px solid #ddd; /* Gray border */
+  border-radius: 4px;  /* Rounded border */
+  padding: 5px; /* Some padding */
+  width: 300px; /* Set a small width */
+}
+
+/* Add a hover effect (blue shadow) */
+
+
+
+.illustration-container {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+
+.illustration-container img {
+  width: 100%;
+  height: auto;
+  max-width: 1920px;
+  padding: 0%;
+}
+
+.form-inline {
+  display: flex;
+  flex-direction:row wrap;
+  align-items: center;
+  width: 50%;
+}
+
+
+input[type=text] , input[type=email]{
+  width: 90%;
+  padding: 10px;
+  margin: 5px 0px 22px 0px;
+  border: none;
+  background: #d8d8d4;
+  border-radius: 8px;
+}
+
+input[type=text]:focus , input[type=email]:focus {
+  background-color: #ddd;
+  outline: none;
+}
+
+button {
+  background-color:cornflowerblue;
+  color: white;
+  padding: 14px 20px;
+  margin: 8px 0;
+  border: none;
+  cursor: pointer;
+  width: 150px;
+  opacity: 1;
+  border-radius: 8px;
+}
+
+button:hover {
+  opacity: .6;
+  transition: .3s;
+}
+
```

Comment:
There's some styling elements here I haven't tried before which sounds really interesting! Will try it out on my next project.